utils/configuration_test.go

```diff
@@ -178,34 +178,22 @@ func TestParseInvalidStorageNoDBSource(t *testing.T) {
 		`{"storage": {"backend": "%s"}}`,
 		`{"storage": {"backend": "%s", "db_url": ""}}`,
 	}
-	for _, backend := range []string{MySQLBackend, SqliteBackend} {
+	for _, backend := range []string{notary.MySQLBackend, notary.SQLiteBackend} {
 		for _, configJSONFmt := range invalids {
 			configJSON := fmt.Sprintf(configJSONFmt, backend)
-			_, err := ParseStorage(configure(configJSON),
-				[]string{MySQLBackend, SqliteBackend})
+			_, err := ParseSQLStorage(configure(configJSON))
 			require.Error(t, err, fmt.Sprintf("'%s' should be an error", configJSON))
 			require.Contains(t, err.Error(),
 				fmt.Sprintf("must provide a non-empty database source for %s", backend))
 		}
 	}
 }
 
-// If a memory storage backend is specified, no DB URL is necessary for a
-// successful storage parse.
-func TestParseStorageMemoryStore(t *testing.T) {
-	config := configure(`{"storage": {"backend": "MEMORY"}}`)
-	expected := Storage{Backend: MemoryBackend}
-
-	store, err := ParseStorage(config, []string{MySQLBackend, MemoryBackend})
-	require.NoError(t, err)
-	require.Equal(t, expected, *store)
-}
-
 // A supported backend with DB source will be successfully parsed.
-func TestParseStorageDBStore(t *testing.T) {
+func TestParseSQLStorageDBStore(t *testing.T) {
 	config := configure(`{
 		"storage": {
-			"backend": "MySQL",
+			"backend": "mysql",
```

Comment:
Non-blocking - do we specifically want different casings of "mysql" (such as "MySQL") to be invalid?  I'm fine with this, although I can see people using the normal casing of MySQL (but certainly "mYSqL" would be weird).